tlslite/keyexchange.py

```diff
@@ -229,40 +229,41 @@ def makeClientKeyExchange(self):
         clientKeyExchange.createRSA(self.encPremasterSecret)
         return clientKeyExchange
 
-# the DHE_RSA part comes from IETF ciphersuite names, we want to keep it
-#pylint: disable = invalid-name
-class DHE_RSAKeyExchange(KeyExchange):
-    """
-    Handling of ephemeral Diffe-Hellman Key exchange
 
+class ADHKeyExchange(KeyExchange):
+    """
+    Handling of anonymous Diffe-Hellman Key exchange
+    FFDHE without signing serverKeyExchange useful for anonymous DH
     NOT stable API, do NOT use
```

Comment:
two things, the summary of the documentation string (the first line) should be a single line and be separated by an empty line from the rest of the doc string

while we're editing it, I think we can remove the "Not stable API" warning, it's not the case any more (it was added when the code was in a different module actually)